build-coordinator/src/main/java/org/jboss/pnc/coordinator/builder/datastore/DatastoreAdapter.java

```diff
@@ -72,7 +73,9 @@ public BuildConfigSetRecord saveBuildConfigSetRecord(BuildConfigSetRecord buildC
      * @return The latest revision of the given build configuration
      */
     public BuildConfigurationAudited getLatestBuildConfigurationAudited(Integer buildConfigurationId) {
-        return datastore.getLatestBuildConfigurationAudited(buildConfigurationId);
+        BuildConfigurationAudited buildConfigAudited = datastore.getLatestBuildConfigurationAudited(buildConfigurationId);
+        buildConfigAudited.getProject().getBuildConfigurations().forEach(BuildConfiguration::getId);
```

Comment:
@matejonnet I don't understand. Are their changes in some other PRs and I can rebase on them?